jinahub/encoder/image/requirements.txt

```diff
@@ -0,0 +1,3 @@
+clip==0.2.0
+torch==1.8.1
+numpy==1.20.3
```

Comment:
numpy should not be needed, it is a basic `jina` dependency

Reply:
Also `jina` is missing?

Reply:
I removed numpy and added jina.